tests/playwright/src/ai-lab-extension.spec.ts

```diff
@@ -62,19 +70,49 @@ describe(`AI Lab extension installation and verification`, async () => {
     test(`Install AI Lab extension`, async () => {
       await extensionsPage.installExtensionFromOCIImage(AI_LAB_EXTENSION_OCI_IMAGE);
       await playExpect
-        .poll(async () => await extensionsPage.extensionIsInstalled(AI_LAB_CATALOG_EXTENSION_LABEL), { timeout: 30000 })
+        .poll(async () => await extensionsPage.extensionIsInstalled(AI_LAB_CATALOG_EXTENSION_LABEL), {
+          timeout: 30_000,
+        })
         .toBeTruthy();
     });
   });
   describe(`AI Lab extension verification`, async () => {
-    test(`Verify AI Lab is present in notification bar and open it`, async () => {
-      const aiLabNavBarItem: Locator = navigationBar.navigationLocator.getByLabel(AI_LAB_NAVBAR_EXTENSION_LABEL);
-      await playExpect(aiLabNavBarItem).toBeVisible();
-      await aiLabNavBarItem.click();
+    test(`Verify AI Lab is responsive`, async () => {
+      [page, webview] = await handleWebview();
+      aiLabPage = new AILabPage(page, webview);
+      await aiLabPage.waitForLoad();
     });
-    test(`Verify AI Lab is running`, async () => {
-      const aiLabWebview: Locator = page.getByLabel(AI_LAB_PAGE_BODY_LABEL);
-      await playExpect(aiLabWebview).toBeVisible();
+    test(`Open Recipes Catalog`, async () => {
+      recipesCatalogPage = await aiLabPage.navigationBar.openRecipesCatalog();
+      await recipesCatalogPage.waitForLoad();
+    });
+    test(`Install ChatBot example app`, { timeout: 780_000 }, async () => {
+      const chatBotApp: AILabAppDetailsPage = await recipesCatalogPage.openRecipesCatalogApp(
+        recipesCatalogPage.recipesCatalogNaturalLanguageProcessing,
+        AI_LAB_AI_APP_NAME,
+      );
+      await chatBotApp.waitForLoad();
+      await chatBotApp.startNewDeployment();
     });
   });
 });
+
+async function handleWebview(): Promise<[Page, Page]> {
+  await navigationBar.navigationLocator.getByLabel(AI_LAB_NAVBAR_EXTENSION_LABEL).click({ timeout: 5_000 });
```

Comment:
I thought using label is always preferred if one is provided in DOM

Reply:
it uses the label. `getByRole('role', { name: 'accessible name'})` searches for accessible name, which also include `aria-label`. So we are searching for role and other property, either text or label. Two checks instead of one. Better from my point of view.

Reply:
done